backend/FwLite/LcmCrdt/Objects/JsonPatchChangeExtractor.cs

```diff
@@ -143,6 +143,34 @@ IChange RewriteComplexFormComponents(IList<ComplexFormComponent> components, Com
             yield return rewriteChange;
         }
 
+        foreach (var rewriteChange in patch.RewriteChanges(
```

Comment:
this stuff is now obsolete, it should probably be cleaned up in another PR, but you don't need to touch this file so you can revert the changes done to it